src/AnalyticsClientFactory.php

```diff
@@ -21,18 +21,11 @@ public static function createAuthenticatedGoogleClient(array $config): Google_Cl
     {
         $client = new Google_Client();
 
-        $client->setClassConfig(
-            'Google_Cache_File',
-            'directory',
-            $config['cache_location'] ?? storage_path('app/laravel-google-analytics/google-cache/')
-        );
-
-        $credentials = $client->loadServiceAccountJson(
-            $config['service_account_credentials_json'],
-            'https://www.googleapis.com/auth/analytics.readonly'
-        );
-
-        $client->setAssertionCredentials($credentials);
+        $client->setScopes([
```

Comment:
Hi, I added back the support to caching, google api advises using [stash](https://github.com/google/google-api-php-client/blob/master/src/Google/Client.php#L1006) but I think we may switch to flysystem since it is already is a laravel dependency, but I'd still have to add a new dependency to your library [php-cache/filesystem-adapter](https://github.com/php-cache/filesystem-adapter)